docs/dcp2_system_design.rst

```diff
@@ -561,31 +561,51 @@ name. ::
 
     day = [0-9]{2}
 
-    qualifier = [a-zA-Z][a-zA-Z0-9]{0,13}
+    qualifier = [a-zA-Z][a-zA-Z0-9]{0,15}
 
     deployment = "dev" | "staging" | "prod"
 
-    project_id = [0-9a-f]{8}(-[0-9a-f]{4}){3}-[0-9a-f]{12}
+    project_id = [0-9a-f]{32}
+
+The ``project_id`` is the project UUID as allocated by the Ingest component. TDR
+does not allow dashes in dataset or snapshot names, so the ``project_id``
+field deviates from the standard string representation of a UUID by omitting the
+dashes.
+
+|nn| The standard representation can be reconstituted by inserting dashes as
+illustrated by the following Python code fragment::
+
+    '-'.join([x[0:8], x[8:12], x[12:16], x[16:20], x[20:36]])
```

Comment:
I think there is a typo here, as UUIDs should only contain 32 chars. I would suggest either of these:

```suggestion
    '-'.join([x[0:8], x[8:12], x[12:16], x[16:20], x[20:]])
```
or
```suggestion
    '-'.join([x[0:8], x[8:12], x[12:16], x[16:20], x[20:32]])
```

Reply:
Fixed.